CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 #### Upcoming Changes
 
+* feat: Implement `SECP related` hints [#1829](https://github.com/lambdaclass/cairo-vm/pull/1829)
+
 * chore: bump pip `cairo-lang` 0.13.3 [#1884](https://github.com/lambdaclass/cairo-vm/pull/1884)
 
+
```

Comment:
Please remove this newline

Reply:
Resolved by https://github.com/lambdaclass/cairo-vm/pull/1829/commits/ef1327370fe871b32cb0393aeaccb9d14fa155c4